caramel/models.py

```diff
@@ -1,21 +1,12 @@
 #! /usr/bin/env python
 # vim: expandtab shiftwidth=4 softtabstop=4 tabstop=17 filetype=python :
 
-# Make things as three-ish as possible (requires python >= 2.6)
-from __future__ import (unicode_literals, print_function,
-                        absolute_import, division)
-# Namespace cleanup
-del unicode_literals, print_function, absolute_import, division
-
-#
-# ----- End header -----
-#
-
 import sqlalchemy as _sa
+from sqlalchemy import func
```

Comment:
all other imports use _ prefix,  since it's not going to be used that much, better to use _sa.func() later than import it as it's own namespace.